Misc/NEWS.d/next/Library/2024-11-27-23-29-05.gh-issue-109798.OPj1CT.rst

```diff
@@ -0,0 +1,2 @@
+Update error messages to be the same in :mod:`datetime`. Patch by Semyon
+Moroz.
```

Comment:
Can you word the NEWS entry so it more clearly expresses the user visible change?

Reply:
@erlend-aasland I corrected the NEWS entry, but I'm not sure if the description is good enough, could you check?